src/sqlite/deviceInfo.ts

```diff
@@ -0,0 +1,37 @@
+import { generate } from 'shortid';
+import { db, getAsync, runAsync } from './index';
+
+const ANONYMOUS_ID_FIELD = "anonymousId";
+
+export const addDeviceInfo = async (): Promise<void> => {
+    const anonymousId = generate();
+    return new Promise(async (resolve) => {
+        const insertSQL = `
+          INSERT OR IGNORE INTO deviceInfo (
```

Comment:
So it will be inserted once, and it will ignore all the other inserts, right?
And only once DB wiped / upticked, new deviceId will be generated and inserted.

Is this logic expected? (just double-checking)